test/run_e2e_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script sources the built ROS2 workspace, installs the necessary Python dependencies,
+# and executes the python end to end tests for the system_metrics_collector package.
+
```

Comment:
Please some more doc around the fact that the objective is for this script to be able to run independently tests on APT packages (canary), and locally. I.e. Emerson was confused, it'd be good to guide the reader.